app/assets/javascripts/float-kit/addon/services/menu.js

```diff
@@ -1,14 +1,13 @@
 import { tracked } from "@glimmer/tracking";
 import { getOwner } from "@ember/application";
 import { action } from "@ember/object";
-import { guidFor } from "@ember/object/internals";
+import { schedule } from "@ember/runloop";
 import Service from "@ember/service";
+import { TrackedArray } from "@ember-compat/tracked-built-ins";
 import DMenuInstance from "float-kit/lib/d-menu-instance";
-import { updatePosition } from "float-kit/lib/update-position";
 
 export default class Menu extends Service {
-  @tracked activeMenu;
-  @tracked portalOutletElement;
+  @tracked registeredMenus = new TrackedArray();
```

Comment:
Does this need to be a `TrackedArray`? I can only see wholesale reassignments of `registeredMenus` prop.